### PR TITLE
WT-5284 Test eviction writing the correct values to data store and lookaside

### PR DESCRIPTION
**Pull request by quchenhao**
Depends on WT-5361 and WT-4683

### CODE REVIEW

File: test/suite/test_las08.py [hunk 1]
```diff
@@ -0,0 +1,158 @@
...
+import unittest, wiredtiger, wttest
+from wiredtiger import stat
+from wtscenario import make_scenarios
+
+def timestamp_str(t):
+    return '%x' % t
+
+# test_las08.py
+# Verify that we write the newest committed version to data store and the
+# second newest committed version to history store.
```
Comment (quchenhao) [inline test/suite/test_las08.py]: Add another test case to test the onpage value follows a tombstone

File: test/suite/test_las08.py [hunk 2]
```diff
@@ -0,0 +1,158 @@
...
+
+        # Prepare some updates
+        self.session.begin_transaction()
+        for i in range(1, 11):
+            cursor[i] = value3
+        self.session.prepare_transaction('prepare_timestamp=' + timestamp_str(5))
+
+        self.check_ckpt_las(value2, value1)
+
+    @unittest.skip("Temparorily disabled until reading time pairs from cells implemented")
```
Comment (quchenhao) [inline test/suite/test_las08.py]: Try to enable this test.

File: test/suite/test_las08.py [hunk 3]
```diff
@@ -0,0 +1,158 @@
...
+        session = self.conn.open_session(self.session_config)
+        session.checkpoint()
+        # Check the data file value
+        cursor = session.open_cursor(self.uri, None, 'checkpoint=WiredTigerCheckpoint')
+        for _, value in cursor:
+            self.assertEqual(value, expected_data_value)
+        cursor.close()
+        # Check the history file value
+        cursor = session.open_cursor("file:WiredTigerLAS.wt", None, 'checkpoint=WiredTigerCheckpoint')
+        for _, key, _, _, _, _, _, _, type, value in cursor:
```
Comment (quchenhao) [inline test/suite/test_las08.py]: Compare timestamp.

**Comment by tetsuo-cpp** [comment]
@quchenhao Is this ready to be looked at?

**Comment by quchenhao** [comment]
@tetsuo-cpp Not yet. The test is failing on windows. More investigation required. Possibly related to WT-5368.

**Comment by quchenhao** [comment]
Windows tests are passing after rebasing on WT-5320. Will wait it to be merged.

**Comment by quchenhao** [comment]
@bvpvamsikrishna @tetsuo-cpp The tests are now passing. Ready for review.

File: test/suite/test_las08.py [hunk 4]
```diff
@@ -0,0 +1,197 @@
...
+        ('string', dict(key_format='S')),
+    ]
+    scenarios = make_scenarios(key_format_values)
+
+    def create_key(self, i):
+        if self.key_format == 'S':
+            return str(i)
+        return i
+
+    def check_ckpt_las(self, expected_data_value, expected_las_value, expected_start_ts, expected_stop_ts):
```
Comment (tetsuo-cpp) [inline test/suite/test_las08.py]: Nit: It's a bit verbose but could you do `expected_start_ts => expected_las_start_ts`?
I got a bit confused as to why the input args were for the second latest version.

**Comment by agorrod** [comment]
@bvpvamsikrishna looks like this is waiting on your review.

**Pull request merged**